ArcGISRuntimeSDKQt_QMLSamples/Layers/GroupLayers/GroupLayers.qml

```diff
@@ -138,15 +139,15 @@ Rectangle {
                     CheckBox {
                         id: parentBox
                         text: name
-                        checked: true
+                        checked: layerVisible
                         onCheckedChanged: layerVisible = checked
                     }
 
                     Repeater {
                         property var groupLyr: scene.operationalLayers.get(layerVisibilityListView.currentIndex)
                         model: groupLyr ? groupLyr.layers : null
-                        delegate: CheckBox {
-                            checked: true
+                        delegate: RadioDelegate {
+                            checked: index == 0
```

Comment:
In QML/JS, always use ===

https://stackoverflow.com/questions/15645705/why-use-instead-of-when-comparing-a-string